CMakeLists.txt

```diff
@@ -152,6 +152,7 @@ option(LWS_WITH_LIBEV "Compile with support for libev" OFF)
 option(LWS_WITH_LIBUV "Compile with support for libuv" OFF)
 option(LWS_WITH_LIBEVENT "Compile with support for libevent" OFF)
 option(LWS_WITH_GLIB "Compile with support for glib event loop" OFF)
+option(LWS_WITH_SDEVENT "Compile with support for systemd event loop" OFF)
```

Comment:
not sure, I wanted to keep the wording aligned with libuv/libevent/glib above

Reply:
Yes it is the only one. I only think that we could easily miss the `d` in systemd and then think about raw Linux event loop or something like that. `sd-event` ist much more specific. Maybe we should use only `Compile with support for sd-event` instead of `systemd event loop`

Reply:
Changed text to "Compile with support for sd-event"